src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -135,113 +135,53 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchFeedItems();
     };
 
-    tasksApiNew = {
```

Comment:
delete the "tasksApiNew" object and promote the functions to top-level scope in the component